ivadomed/main.py

```diff
@@ -35,13 +35,20 @@ def get_parser():
                                     ' The parameter indicates the number of 2D slices used to generate GIFs, one GIF '
                                     'per slice. A GIF shows predictions of a given slice from the validation '
                                     'sub-dataset. They are saved within the log directory.')
+    optional_args.add_argument('-t', '--thr_increment', required=False, type=float,
```

Comment:
i would suggest to use `--thr-increment`

Reply:
Done in [b44b8be](https://github.com/ivadomed/ivadomed/pull/383/commits/b44b8bef5c948ab3f88f3d59b6af2a84ce1d1b3f)